frontend/src/pages/distributedWorkloads/global/projectMetrics/GlobalDistributedWorkloadsProjectMetricsTab.tsx

```diff
@@ -8,43 +18,41 @@ import { WorkloadResourceMetrics } from './sections/WorkloadResourceMetrics';
 import { DWSectionCard } from './sections/DWSectionCard';
 
 const GlobalDistributedWorkloadsProjectMetricsTab: React.FC = () => {
-  const { projectCurrentMetrics } = React.useContext(DistributedWorkloadsContext);
+  const { clusterQueue, localQueues } = React.useContext(DistributedWorkloadsContext);
+  const requiredFetches = [clusterQueue, localQueues];
+  const error = requiredFetches.find((f) => !!f.error)?.error;
+  const loaded = requiredFetches.every((f) => f.loaded);
 
-  if (projectCurrentMetrics.error) {
+  if (error) {
     return (
       <EmptyStateErrorMessage
         title="Error loading distributed workload metrics"
-        bodyText={projectCurrentMetrics.error.message}
+        bodyText={error.message}
       />
     );
   }
 
-  if (!projectCurrentMetrics.loaded) {
+  if (!loaded) {
     return (
       <Bullseye style={{ minHeight: 150 }}>
         <Spinner />
       </Bullseye>
     );
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { getWorkloadCurrentUsage, topResourceConsumingWorkloads } = projectCurrentMetrics;
-  // eslint-disable-next-line no-console
-  console.log({ topResourceConsumingWorkloads });
-
-  //TODO: need 'no quota' logic
-  // if (false) {
-  //   return (
-  //     <EmptyState>
-  //       <EmptyStateHeader
-  //         titleText="Quota is not set"
-  //         headingLevel="h4"
-  //         icon={<EmptyStateIcon icon={WrenchIcon} />}
-  //       />
-  //       <EmptyStateBody>Select another project or set the quota for this project.</EmptyStateBody>
-  //     </EmptyState>
-  //   );
-  // }
+  // clusterQueue.data will only be defined here if it has spec.resourceGroups (see DistributedWorkloadsContext)
+  if (!clusterQueue.data || localQueues.data.length === 0) {
```

Comment:
For one -- Don't describe your data outside of its definition -- this will quickly get out of date.

For two -- Why is clusterQueue important here? If you don't have a local queue, doesn't that invalidate the project?

Reply:
One -- maybe this comment just isn't useful anyway but what it describes is obvious from the code where it is defined. The point of this comment was "the design calls for this empty state if the cluster queue has no resource groups, that case is covered here even though it looks like we're only checking for it to exist". Confusing and probably unnecessary now that I return to it.

Two -- it's possible for a user to create a ClusterQueue and LocalQueue but not have any `resourceGroups` defined on the ClusterQueue, which is what sets the resource quotas.

Maybe we should be checking spec.resourceGroups here directly instead of in the context to resolve both of these?

Reply:
Perhaps this is a DW thing I don't understand... how is
```
          titleText="Quota is not set"
```
valid when you don't have resources? 🤔 Are Quotas set inside resources?

LocalQueue is about having none... where ClusterQueue is about having no resources? 🤔 This feels weird.

Reply:
You need both Cluster and Local. ClusterQueue defines the available resources for the whole cluster, LocalQueue defines some subset of those resources to claim (typically one per namespace, but you can have more than one). I think the idea is to cover this in product documentation initially, but one of the future epics is to provide a UI settings page for configuring these quotas and add a button here to take you there.

Reply:
although hang on, checking my assumption here... a localQueue doesn't have resourceRequests in its spec. I think you just need to have one existing to be ready to start workloads and that got lumped into the "set a quota" logic. I was just implementing this based on what @astefanutti asked for

Reply:
> Are Quotas set inside resources?

My understanding is that the "Quota" is the available resources in `clusterQueue.spec.resourceRequests`